ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/css/formcontainerview.css

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  ******************************************************************************/
 
-[data-cmp-visible="false"] {
+.cmp-adaptiveform-container:not(.cq-Editable-dom) > form.cmp-adaptiveform-container [data-cmp-visible="false"] {
```

Comment:
is a wrapper div added by container always. I think there was a mechanism to get rid of that wrapper div (in which case this will fail).

Reply:
Why is runtime client library referring editable CSS selector ?